overlays.py

```diff
@@ -0,0 +1,126 @@
+
+class ShortcutTableItem(QTableWidgetItem):
+
+    def __init__(self):
+        super().__init__()
+        # disable selection, editing
+        self.setFlags(QtCore.Qt.NoItemFlags)
+
+
+class ShortcutsOverlay(QWidget):
+    shortcuts = {"Color By": [("Cell", "Alt+C"),
+                              ("Material", "Alt+M"),
+                              ("Temperature", "Alt+T"),
+                              ("Density", "Alt+D")],
+                 "View Options": [("Apply Changes", c_key + "+Enter"),
+                                  ("Undo", c_key + "+Z"),
+                                  ("Redo", "Shift+Ctrl+Z"),
+                                  ("Restore Default Plot", c_key + "+R"),
+                                  ("Zoom", "Alt+Shift+Z"),
+                                  ("Zoom", "Shift+scroll"),
+                                  ("Toggle Masking", c_key + "+M"),
+                                  ("Toggle Highlighting", c_key + "+L"),
+                                  ("Set XY Basis", "Alt+X"),
+                                  ("Set YZ Basis", "Alt+Y"),
+                                  ("Set XZ Basis", "Alt+Z"),
+                                  ("Update Plot Origin", "Double-click"),
+                                  ("Open Context Menu", "Right-click")],
+                 "Menus": [("Hide/Show Options Dock", c_key + "+D"),
+                           ("Save View", c_key + "+S"),
+                           ("Open View", c_key + "+O"),
+                           ("Save Plot Image", c_key + "+Shift+S"),
+                           ("Quit", c_key + "+D"),
+                           ("Display Shortcuts", "?")]}
+
+    # colors
+    header_color = QtGui.QColor(150, 150, 150, 255)
+    fillColor = QtGui.QColor(30, 30, 30, 200)
+    framePenColor = QtGui.QColor(255, 255, 255, 120)
+    textPenColor = QtGui.QColor(152, 196, 5, 255)
+
+    def __init__(self, parent):
+        super().__init__(parent)
+
+        # transparent window fill
+        self.layout = QVBoxLayout()
+        self.setLayout(self.layout)
+
+        n_rows = max((len(scts) for scts in self.shortcuts.values()))
+        n_rows += 1  # plus one for header
+        n_cols = len(self.shortcuts.keys()) * 3
+        self.tableWidget = QTableWidget(n_rows, n_cols, self)
+        self.layout.addWidget(self.tableWidget)
+
+        # set all items to a non-editable cell item
+        for i in range(n_rows):
+            for j in range(n_cols):
+                self.tableWidget.setItem(i, j, ShortcutTableItem())
+
+        self.tableWidget.setShowGrid(False)
+        self.tableWidget.setSizePolicy(QSizePolicy.Expanding,
+                                       QSizePolicy.Expanding)
+        self.tableWidget.verticalHeader().setVisible(False)
+        self.tableWidget.horizontalHeader().setVisible(False)
+        self.tableWidget.setStyleSheet("background-color:" \
+                                       "rgba(30, 30, 30, 230);" \
```

Comment:
What I meant was:
```Python
        self.tableWidget.setStyleSheet("background-color: rgba(30, 30, 30, 230);"
                                       "border: 0px;"
                                       "padding: 20px")
```

Reply:
Ah, sorry! Meant to clarify and ended up pushing that change instead. Updating...